types/suggestions.d.ts

```diff
@@ -3,4 +3,26 @@ export interface Suggestion {
   toString(): string
 }
 
-export function getSuggestedQuery(element: HTMLElement): Suggestion | undefined
+export type Variant =
+  | 'get'
+  | 'getAll'
+  | 'query'
+  | 'queryAll'
+  | 'find'
+  | 'findAll'
+
+export type Method =
+  | 'Role'
+  | 'LabelText'
+  | 'PlaceholderText'
+  | 'Text'
+  | 'DisplayValue'
+  | 'AltText'
+  | 'Title'
+  | 'TestId'
```

Comment:
This is incompatible with your change from #628? :innocent:

Reply:
As for the other thread, you will receive an error so is simpler than with javascript to understand that you have to pass `Role` instead of `role`.

Reply:
So we configure typescript to only accept `Role`, but at the same time we make javascript checks case insensitive by adding two `toLowerCase()` calls?

It's not a huge issue. Just feels wrong to me.

Reply:
FWIW, I'm fine with it, though I think it would be better to allow the lower case version here as well.